WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,8 @@
 RSpec.configure do |config|
   config.before(:suite) do
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 0;")
     DatabaseCleaner.clean_with :truncation
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 1;")
```

Comment:
I've just removed these and everything works fine. I can't issue a truncate command in MySQL on a table with foreign keys, and I thought this gave an error previously but it seems it's not necessary. I'll get rid of it.